site/docs/v1/tech/apis/identity-providers/_oauth-idp-request-body.adoc

```diff
@@ -7,6 +7,11 @@ The configuration for each Application that the identity provider is enabled for
 
 [field]#identityProvider.applicationConfiguration``[applicationId]``.buttonText# [type]#[String]# [optional]#Optional#::
 This is an optional Application specific override for the top level [field]#buttonText#.
+ifeval::["{idp_type}" == "Google"]
++
+[since]#Since 1.44.0#
+The [field]#identityProvider.applicationConfiguration``[applicationId]``.buttonText# value will not be used when [field]#identityProvider.loginMethod# is set to `UsePopup` or `UseVendorJavaScript` because of changes to the Google JavaScript API.
```

Comment:
A general comment on the ".. because of the changes to the Google JavaScript API." 

This sort of sounds like we are blaming them for something. 

Could we instead use wording such 

> The ... buttonText value may only be used when the loginMethod is set to `UseRedirect`. The button text is managed by Google for all other configurations. Google allows for minor adjustments using their API. You may specify any valid API parameter in the [field]#identityProvider.properties.button#  config. 

Or something along those lines.